crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -59,6 +61,10 @@ impl Violation for Airflow3Removal {
             }
         }
     }
+
+    fn fix_title(&self) -> Option<String> {
+        Some("Replace deprecated keywords in Airflow 3.0".to_string())
+    }
```

Comment:
We should return `None` if there's no replacement (fix). It's otherwise confusing when we suggest that they replace the deprecated keyword but there's no replacement because it was removed.

Reply:
Thanks, done.